src/commands/cmd_query.c

```diff
@@ -61,12 +61,20 @@ void Graph_Query(void *args) {
 
 	QueryCtx_BeginTimer(); // Start query timing.
 
+	cypher_parse_result_t *params_parse_result = parse_params(command_ctx->query);
```

Comment:
This area should have more comprehensive comments to describe which query parts are being parsed at which point.